app/police-custody-form/index.ts

```diff
@@ -0,0 +1,22 @@
+import { Request, Response, Router } from 'express'
+import { addEvents } from './controllers'
+import { BasmRequest } from '../../common/types/basm_request'
+import { BasmResponse } from '../../common/types/basm_response'
+
+const router = Router({ mergeParams: true })
+
+router.get('/', (req: Request, res: Response) => {
+  const basmRes = res as BasmResponse
+  delete basmRes.breadcrumb
+  basmRes.render('police-custody-form/police-custody-form')
+})
+
+router.post('/', (req: Request, res: Response) => {
+  const basmReq = req as unknown as BasmRequest
+  addEvents(basmReq, res as BasmResponse)
+})
+
+module.exports = {
```

Comment:
We should probably do `export` instead of `module.exports =`, to keep it consistent with the other files

Reply:
Updated.